website/docs/how-to/how-to-add-sso-azure-saml.md

```diff
@@ -0,0 +1,105 @@
+Click save. 
+
+![Azure: SSO Setup Overview](/img/sso-azure-saml-azure-details-overview.png)
+![Azure: SSO Setup Section 1](/img/sso-azure-saml-section-one.png)
+
+**d) Section 2: Attributes & Claims**
+
+Unleash requires an email to be sent from the SSO provider so make sure to set the "Unique User Identifier" to the "user.mail" source attribute.
+
+In addition you may provide the following attributes:
+
+- firstName
+- lastName
+
+_(These will be used to enrich the user data in Unleash, but are not required)._
+
+![Azure: SAML Unique Identifier](/img/sso-azure-saml-unique-id-email-id.png)
+![Azure: SAML Attributes](/img/sso-azure-saml-attributes-claim.png)
+
+> Please make sure to replace URLs with the public URL for your Unleash instance. This will require correct region prefix and the instance name. 
+>
+> The correct format is: https://**[region]**.app.unleash-hosted.com/**[instanceName]**/auth/saml/callback
+
+**e) Get the Azure AD Setup Details**
+
+Make note of the following details necessary for the Unleash SAML configuration.
+
+Unleash will need:
+- Azure AD Identifier
+- Login URL
+- X.509 Certificate (in the Federation Metadata XML)
+
+![Azure: Azure Details for Unleash](/img/sso-azure-saml-azure-details.png)
+![Azure: Azure X509 for Unleash](/img/sso-azure-saml-x509cert.png)
+
+### Step 4: Configure SAML 2.0 provider in Unleash {#step-4}
+
+Go back to Unleash Admin Dashboard and navigate to `Admin Menu -> Single-Sign-On -> SAML`. Fill in the values captured in the _"Get the Azure AD Setup Details"_ step.
+This is how the Azure details map to Unleash
+- Azure AD Identifier > Entity ID (should be formatted as https://sts.windows.net/**{GUID}**)
+- Login URL > Single Sign-On URL (should be formatted as https://login.microsoftonline.com/**{GUID}**/saml2)
+- Azure Federation Metadata XML (copy the X509Certificate) > X.509 Certificate
+
+You may also choose to “Auto-create users”. This will make Unleash automatically create new users on the fly the first time they sign-in to Unleash with the given SSO provider (JIT). If you decide to automatically create users in Unleash you must also provide a list of valid email domains separated by commas. You must also decide which global Unleash role they will be assigned (Editor role will be the default).
+
+![Unleash: SAML 2.0](/img/sso-azure-saml-unleash-config.png)
+
+### Step 5: Validate {#step-5}
+
+You have now successfully configured Unleash to use SAML 2.0 together with Azure AD as an IdP. Please note that you also must assign users to the application defined in Azure AD to actually be able to log-in to Unleash.
+
+Try signing out of Unleash. If everything is configured correctly you should be presented with the option to sign in with SAML 2.0.
+
+Additionally, Azure gives the option to test logging in with a user at the bottom of the Azure AD SAML setup page. 
+
+![Azure: Test User](/img/sso-azure-saml-test-user.png)
```

Comment:
This isn't really a step, so let's remove that bit of the heading.

Also, the thing about assigning users in Azure seems like a pretty major point. Maybe that should be it's own step? 

```suggestion
### Step 4: Add users {#step-4}

You must **assign users to the application you defined in Azure AD** for them to be able to log in.

[... maybe add some steps here?]

### Validation

If everything is set up correctly, you should now be able to sign in with the SAML 2.0 option. You can verify that this works by logging out of Unleash: the login screen should give you the option to sign in with SAML 2.0.

You can also test the integration in Azure by using the "test single sign on" step in the SAML setup wizard. 

<Figure caption="The SAML setup wizard contains a step that lets you test your SAML 2.0 integration. You can use this to verify that everything is configured correctly within Azure." img="/img/sso-azure-saml-test-user.png"/>
```

Reply:
@thomasheartman I've made lots of updates to this, let me know what you think!